### PR TITLE
Add the Swan Lake Alpha5 Release Note

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the Swan Lake Alpha5 release note.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,401 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+### Overview of Ballerina Swan Lake Alpha5
+
+<em>This is the <fifth> Alpha release in a series of planned Alpha and Beta releases leading up to the Ballerina Swan Lake GA release.</em>
```
Comment (dakshitha) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:7]:
```suggestion
<em>This is the fifth Alpha release in a series of planned Alpha and Beta releases leading up to the Ballerina Swan Lake GA release.</em> 
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+### Standard Library Updates
+
+#### New Features
+
+##### GraphQL Package
```
Comment (dakshitha) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:308]: Shall we stick to the same styling format that we used for package names in the Alpha4 release note?

**Review by dakshitha — COMMENTED** [review]
Please check the comments in-line.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+address.country.name="USA"
+
+[[personArray]]
+name = "hinduja"
+id = 12
+address.city="London"
+address.country.name="UK"
+```
+
+###### Support for Configurable Variables with Multidimentional Arrays
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:160]:
```suggestion
###### Support for Configurable Variables with Multidimensional Arrays
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+```ballerina
+import main.foo;
+import importedOrg/mod;
+configurable string mainVar = ?;
+public function main() {
+// use imported modules
+}
+```
+
+The`foo.bal` file of the `main.foo` module will be as follows.
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:192]:
```suggestion
The `foo.bal` file of the `main.foo` module will be as follows.
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 5]
```diff
@@ -0,0 +1,400 @@
...
+}
+```
+
+```bash
+bal run file.bal -- --name riyafa  100 --score=99.9 Good
+```
+
+This example, which is the same as above includes `100`, which gets mapped to `efficiency`, and `Good`, which gets mapped to `character` as operands. 
+
+Both operand and option parameters can be of types int, float, decimal, string, array of any of these types, and union of any of these types with nil.
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:265]:
```suggestion
Both operand and option parameters can be of types `int`, `float`, `decimal`, `string`, array of any of these types, and union of any of these types with `nil`. 
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 6]
```diff
@@ -0,0 +1,400 @@
...
+##### UDP Package
+Added compiler plugin validation for UDP services.
+
+##### HTTP Package
+Added compiler plugin validations for HTTP services.
+
+##### WebSocket Package
+Added compiler plugin validation for WebSocket services.
+
+##### Websub Package
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:327]:
```suggestion
##### WebSub Package
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 7]
```diff
@@ -0,0 +1,400 @@
...
+Added compiler plugin validation for WebSub subscriber services.
+
+##### WebsubHub Package
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:330]:
```suggestion
##### WebSubHub Package
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 8]
```diff
@@ -0,0 +1,400 @@
...
+### Developer Tools Updates
+
+#### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake Alpha5 of the repositories below.
+
+- [Dev Tools](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FDevTools)
+- [Language Server](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FLanguageServer)
+
+
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:400]: Extran new lines

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 9]
```diff
@@ -0,0 +1,400 @@
...
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
+### Runtime Updates
+
+#### New Features
+
+###### Support for Configurable Variables with Records Having Fields of Record Types
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:104]: Shouldn't this be a level 5 heading?

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 10]
```diff
@@ -0,0 +1,400 @@
...
+```toml
+[person]
+name = "waruna"
+id = 10
+address.city="Colombo"
+address.country.name="Sri Lanka"
+
+```
+
+###### Support for Configurable Variables with Arrays Having Fields of Record Types
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:138]: Shouldn't this be a level 5 heading?

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 11]
```diff
@@ -0,0 +1,400 @@
...
+configurable int[][] & readonly int2DArr = ?;
+```
+
+The  `Config.toml` would be as follows.
+
+```toml
+int2DArr = [[1,2],[3,4]]
+```
+
+###### Support for Optional Module Name in TOML Syntax of Configurable Variables
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:172]: Shouldn't this be a level 5 heading? Check all.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 12]
```diff
@@ -0,0 +1,400 @@
...
+mainVar = "variable from root module"
+[main.foo]
+fooVar = "variable from non-root module of the root package"
+[importedOrg.mod]
+modVar = "variable from non-root package"
+```
+
+#### Improvements
+
+###### Improved Command-Line Argument Parsing
```
Comment (anupama-pathirage) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:217]: Shouldn't this be a level 5 heading?

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 2 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 1 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 7 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md]: Changed to "`websubhub` package" as per https://github.com/ballerina-platform/ballerina-dev-website/pull/2113#discussion_r618606537.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 6 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md]: Changed to "`websub` Package" as per https://github.com/ballerina-platform/ballerina-dev-website/pull/2113#discussion_r618606537

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 8 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 9 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:104]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 10 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:138]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 11 above)
Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:172]: Fixed.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 13]
```diff
@@ -0,0 +1,398 @@
...
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+### Standard Library Updates
+
+#### New Features
+
+##### `graphql`` Package
```
Comment (dakshitha) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:308]:
```suggestion
##### `graphql` Package
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 14]
```diff
@@ -0,0 +1,398 @@
...
+
+- Added named fragment support
+- Added enum support
+- Added map support
+- Added compiler plugin validation for GraphQL services
+
+##### `email` Package
```
Comment (dakshitha) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:315]: Can't we add a single line for this? e.g.

Added compiler plugin validation for services in the following packages: `udp`, `http`, `websocket`, ... and `tcp`.

Comment (praneesha) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:315]: @ldclakmal

Comment (shafreenAnfar) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:315]: Yeah, having a single line would be better.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 15]
```diff
@@ -0,0 +1,398 @@
...
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+### Standard Library Updates
+
+#### New Features
+
+##### `graphql` Package
+
+- Added named fragment support
```
Comment (ThisaruGuruge) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:310]:
```suggestion
- Added named fragment support in GraphQL queries.
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 16]
```diff
@@ -0,0 +1,398 @@
...
+- Added enum support
```
Comment (ThisaruGuruge) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:311]:
```suggestion
- Added enum support for GraphQL services.
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 17]
```diff
@@ -0,0 +1,398 @@
...
+- Added map support
```
Comment (ThisaruGuruge) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:312]:
```suggestion
- Added map support for GraphQL fields.
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 18]
```diff
@@ -0,0 +1,398 @@
...
+Added compiler plugin validations for RabbitMQ services.
+
+##### `nats` Package
+Added compiler plugin validations for NATS services.
+
+##### `stan` Package
+Added compiler plugin validations for STAN services.
+
+##### `tcp` Package
+Added compiler plugin validation for TCP services.
```
Comment (ldclakmal) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:346]:
```suggestion
Added compiler plugin validation for services in the following packages: `http`, `grpc`, `websocket`, `websub`, `websubhub`, `graphql`, `kafka`, `rabbitmq`, `nats`, `stan`, `email`, `tcp`, `udp` and `file`

##### `graphql` Package

- Added named fragment support
- Added enum support
- Added map support
```

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 14 above)
Comment (ldclakmal) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md]: Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/2113/commits/99d6ed39eb0c61c5fbe48c180fb843857a5bf056

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md (hunk 18 above)
Comment (ThisaruGuruge) [reply downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md]: Shall we order these alphabetically?
Added compiler plugin validation for services in the following packages: `email`, `file`, `graphql`, `grpc`,  `http`, `kafka`, `nats`, `rabbitmq`, `stan`, `tcp`, `udp`, `websocket`, `websub`, and `websubhub`.

File: downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md [hunk 19]
```diff
@@ -0,0 +1,366 @@
...
+
+#### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Alpha5](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha5%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+### Standard Library Updates
+
+#### New Features
+
+Added compiler plugin validation for services in the following packages: `http`, `grpc`, `websocket`, `websub`, `websubhub`, `graphql`, `kafka`, `rabbitmq`, `nats`, `stan`, `email`, `tcp`, `udp` and `file`
```
Comment (ThisaruGuruge) [inline downloads/swan-lake-release-notes/swan-lake-alpha5/RELEASE_NOTE.md:308]: Order the package names alphabetically.

```suggestion
Added compiler plugin validation for services in the following packages: `email`, `file`, `graphql`, `grpc`, `http`, `kafka`, `nats`, `rabbitmq`, `stan`, `tcp`, `udp`, `websocket, `websub`, and `websubhub`.
```

**Review by sameerajayasoma — APPROVED** [review]
LGTM

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**